.travis.yml

```
@@ -1,6 +1,24 @@
+dist: bionic
 language: go
+go_import_path: github.com/buger/goreplay
 go: 1.12
-script: sudo -E bash -c "source /etc/profile && eval '$(gimme 1.12)' && export GOPATH=$HOME/gopath:$GOPATH && go get && GORACE='halt_on_error=1' go test ./... -v -timeout 120s -race"
+script: sudo -E bash -c "source /etc/profile && eval '$(gimme 1.12)' && export GOPATH=$HOME/gopath:$GOPATH && go get && GORACE='halt_on_error=1' go test ./... -v -timeout 120s -race" || true
```

I don't think it will be good to ignore tests(specifically github checks)
```suggestion
script: sudo -E bash -c "source /etc/profile && eval '$(gimme 1.12)' && export GOPATH=$HOME/gopath:$GOPATH && go get && GORACE='halt_on_error=1' go test ./... -v -timeout 120s -race"
```